assesspy/sales_chasing.py

```diff
@@ -53,49 +55,65 @@ def pct_in_range(x, min, max):
     # Determine what percentage of the data would be within the specified
     # bounds in the ideal distribution
     pct_ideal = pct_in_range(ideal_dist, bounds[0], bounds[1])
-
-    # Determine what percentage of the data is actually within the bounds
     pct_actual = pct_in_range(ratio, bounds[0], bounds[1])
 
-    return pct_actual > pct_ideal
+    return bool(abs(pct_actual - pct_ideal) > gap)
```

Comment:
I added a new requirement that the absolute difference in percentages be greater than some threshold because this method was _waaay_ too sensitive before.